apps/docs/content/guides/local-development.mdx

```diff
@@ -27,6 +27,11 @@ Develop locally while running the Supabase stack on your machine.
     pnpm add supabase --save-dev --allow-build=supabase
     ```
 
+    <Admonition type="note">
+      The `--allow-build=supabase` flag is required pnpm version 10 or higher. If you're using an
+      older version of pnpm, omit this flag.
+    </Admonition>
+
```

Comment:
```suggestion
    <Admonition type="note">
    
    The `--allow-build=supabase` flag is required on pnpm version 10 or higher. If you're using an older version of pnpm, omit this flag.
    
    </Admonition>

```

Not just me being a stickler 😄 omitting the blank lines makes Prettier split the lines which inserts a line break in the final output. A lint rule for this is coming in the next linter release

Reply:
Is there a way to run those locally ? That's always quite tedious to need to push the change before actually getting all those comments on the PR 😓

Reply:
pnpm -F docs run lint:mdx